includes/widgets/icon-box.php

```diff
@@ -404,22 +404,38 @@ protected function register_controls() {
 			]
 		);
 
-		$this->add_control(
+		$this->add_responsive_control(
 			'rotate',
 			[
 				'label' => esc_html__( 'Rotate', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'deg' ],
+				'range' => [
+					'deg' => [
+						'min' => 0,
+						'max' => 360,
+						'step' => 1,
+					],
+				],
 				'default' => [
-					'size' => 0,
 					'unit' => 'deg',
+					'size' => '',
+				],
+				'tablet_default' => [
+					'unit' => 'deg',
+					'size' => '',
+				],
+				'mobile_default' => [
+					'unit' => 'deg',
+					'size' => '',
```

Comment:
@hein-obox Mobile and Tablet defaults aren't enough.
Since we have the Additional Breakpoints experiment - you need to take into account all of the active breakpoints, in case the user activated the experiment and is using any additional breakpoints, such as Laptop.